src/project/manifest/mod.rs

```diff
@@ -1090,12 +1160,21 @@ mod test {
             .join("\n"));
     }
 
-    fn test_remove(file_contents: &str, name: &str, kind: SpecType, platform: Option<Platform>) {
+    fn test_remove(
+        file_contents: &str,
+        name: &str,
+        kind: SpecType,
+        platform: Option<Platform>,
+        feature_name: Option<&FeatureName>,
+    ) {
         let mut manifest = Manifest::from_str(Path::new(""), file_contents).unwrap();
 
+        let f_name = feature_name.unwrap_or(&FeatureName::Default);
```

Comment:
Please just use the full name for local bindings, f is not necessary immediately clear.

Reply:
You are right, is `feature_name_opt` a good name for the option variant?
fixed in: https://github.com/prefix-dev/pixi/pull/680/commits/b67a0e1bcb71d921a1011a7cfee31d95ff13808b